.unreleased/bugfix_6024

```diff
@@ -0,0 +1,3 @@
+Fixes: #6035 UPDATE on compressed chunk crashes server
+
+Thanks: @alexanderlaw
```

Comment:
Could you add a short text to the thank you note? It will be copied into the CHANGELOG.md file and we usually include a short sentence about why we thank the person.

Reply:
Fixed.